Description: test: drop test that depends on Python internals
This test works on Python 2 because for some reason the "Python" lexers
comes later, and overrides the "IPython" one in the mime type index. On
Python 3, that order is different, and a lookup for "text/x-python"
will return the "IPython" lexer instead.
Author: Antonio Terceiro <asa@terceiro.xyz>
Last-Update: 2020-02-13
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/test_pygments.rb
+++ b/test/test_pygments.rb
@@ -185,7 +185,6 @@
   def test_find_lexer_by_mimetype
     assert_equal P::Lexer['Ruby'], P::Lexer.find_by_mimetype('text/x-ruby')
     assert_equal P::Lexer['JSON'], P::Lexer.find_by_mimetype('application/json')
-    assert_equal P::Lexer['Python'], P::Lexer.find_by_mimetype('text/x-python')
   end
 end
 
